server/helper/handler.js

```diff
@@ -4,11 +4,45 @@ var helper = require('./helpers.js');
 
 module.exports = {
   getAllItems: function (req, res) {
-    item.getAll(function (items) {
-      res.send(items);
+    item.getAll(function (err, items) {
+      if(err) {return res.status(400);}
```

Comment:
If status code is set with status() method, it should be sent to client by chaining .send() or .end()
Or more better way is just call [sendStatus()](http://expressjs.com/en/4x/api.html#res.sendStatus)